### ISSUE TITLE
ValueError: Incorrect family code in device address.

### ISSUE THREAD

**Issue by robmarkcole** [adafruit/Adafruit_CircuitPython_DS18X20#14]
Attempting to use the DS18X20 with Metro M4 Airlift Lite with circuitpython 5.0.0-beta.0 with the `ds18x20_simpletest.py `example. I hit [this error](https://github.com/adafruit/Adafruit_CircuitPython_OneWire/issues/16), so am now setting a large `_maximum_devices`. However I now hit a different error - `ValueError: Incorrect family code in device address.`

```
>>> import time
>>> import board
>>> from adafruit_onewire.bus import OneWireBus
>>> from adafruit_ds18x20 import DS18X20
>>> ow_bus = OneWireBus(board.D5)
>>> ow_bus.maximum_devices(200)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'int' object is not callable
>>> ow_bus._maximum_devices = 200
>>> ds18 = DS18X20(ow_bus, ow_bus.scan()[0])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "adafruit_onewire/bus.py", line 156, in scan
RuntimeError: Maximum device count of 200 exceeded.
>>> ow_bus._maximum_devices = 2000
>>> ds18 = DS18X20(ow_bus, ow_bus.scan()[0])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "adafruit_ds18x20.py", line 56, in __init__
ValueError: Incorrect family code in device address.
```

**Comment by ladyada** [comment]
which exact device?

**Comment by robmarkcole** [comment]
18B20 
<img width="430" alt="image" src="https://user-images.githubusercontent.com/11855322/70117127-84db7a80-165c-11ea-84f9-da6a111d42f3.png">

**Comment by ladyada** [comment]
please show an entire wiring diagram including pullup resistor

**Comment by robmarkcole** [comment]
With pull up resistor no errors, all working. This is my fault for not reading the docs, so sorry! Put my writeup on https://github.com/robmarkcole/circuitpython-projects/tree/master/1-wire%20temperature

**Comment by ladyada** [comment]
all good, we'll add a warning : )

**Comment by dhalbert** [comment]
Fixed by #15.
